Lesson24/deniz/newsletter assignment/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Newsletter sign-up form</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="card">
+        <div class="right">
+            <h1>Stay updated!</h1>
+            <p>Join 60,000+ product managers receiving <br>monthly updates on:</p>
+            <ul>
+                <li>Product discovery and building what matters</li>
+                <li>Measuring to ensure updates are a success</li>
+                <li>And much more!</li>
+            </ul>
+            <form>
```

Comment:
Disable default HTML validation as you need to do validation in JS for this assignment
```suggestion
            <form novalidate>
```